.github/ISSUE_TEMPLATE/1-DEV_ISSUE.md

```diff
@@ -1,31 +1,21 @@
 ---
-Title: "issue-<NUMBER>-<TITLE>"
-Name: ⚙️ Internal Development Issue
-About: For tracking technical tasks or bugs
-Labels: ''
-Assignees: ''
-
+name: Development Request
+title: 'issue-<NUMBER>-<TITLE>'
 ---
 
-## **Summary**
-Briefly describe the issue or task.
-
-## **Steps to Reproduce**
-If applicable, list key steps to replicate the issue.
+# Issue
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is.
 
-## **Technical Details**
-Key components/files affected, relevant logs, or errors.
+## Suggested Solution
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
 
-## **Acceptance Criteria**
-- [ ] Criteria 1
-- [ ] Criteria 2
-- [ ] Criteria 3
+## Alternatives Considered
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
```

Comment:
remove, most things on here are meant for newcomers to the repository which doesn't apply to us

Reply:
done!